smsdk/config/api_endpoints.json

```diff
@@ -28,6 +28,10 @@
   },
   "DataViz": {
     "estimate_cycle": "/v1/selector/datavis/estimate/cycle",
-    "estimate_part": "/v1/selector/datavis/estimate/part"
+    "estimate_part": "/v1/selector/datavis/estimate/part",
+    "task": "/v1/datavis/task/async"
+  },
+  "KPI": {
+    "availible_kpis": "/v1/selector/datavis/kpi/y_axis"
```

Comment:
This is the API to get the names of KPIs given a list of machineTypes. Let's use the assets API instead to get the full list of KPIs. In addition to not requiring a payload, this will give us all the kpi info (name, formula, dependencies...), not just the name.

Reply:
assets API is `GET /v1/selector/assets/`